repos/system_upgrade/el7toel8/libraries/config/architecture.py

```diff
@@ -6,8 +6,8 @@
 ARCH_X86_64 = 'x86_64'
 ARCH_ARM64 = 'aarch64'
 ARCH_PPC64LE = 'ppc64le'
-ARCH_S390X = 's390x'
-ARCH_ACCEPTED = (ARCH_X86_64, ARCH_ARM64, ARCH_PPC64LE, ARCH_S390X)
+# ARCH_S390X = 's390x'
```

Comment:
What's the reason for removing the constant here now? and also from the accepted arches. Somehow I don't understand why this change was applied now. I thought we are working on making it work. Some reasoning in the PR and commit message for why to disable something that has been enabled before would be great.

Reply:
`ARCH_ACCEPTED` should be kept unchanged. It doesn't have anything common with the set of supported arches. Instead, should be provided rather new constant: `ARCH_SUPPORTED`.